modules/aggregation-layers/src/hexagon-layer/hexagon-aggregator.js

```diff
@@ -30,16 +30,21 @@ import {createIterable, log} from '@deck.gl/core';
 
  * @return {Object} - hexagons and countRange
  */
-export function pointToHexbin({data, radius, getPosition}, viewport) {
+export function pointToHexbin(opts) {
```

Comment:
pointToHexbin is designed to take `(props, viewport)` changing the function signature here will make it harder to override

Reply:
Same as above comment.

Reply:
resolved this in #3951